kettle/update.py

```diff
@@ -62,21 +62,21 @@ def main():
         mj_ext = ' --reset-emitted'
 
     if os.getenv('DEPLOYMENT', 'staging') == "prod":
-        call(f'{mj_cmd} {mj_ext} --days {DAY} | pv | gzip > build_day.json.gz')
+      call(f'{mj_cmd} {mj_ext} --days {DAY} | jq -c '.[]' | pv | gzip > build_day.json.gz')
```

Comment:
The '' need to be escaped 

`jq -c '\.[]\'`

Reply:
Done